### PR TITLE
:lipstick: Created the booklets page hero section #15

### LINKED ISSUE #15: 💄 🎨 Create the booklets page hero section
#  Create the booklets page hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user I want a nice hero section on the booklets page that captures my attention and introduces the page content. Reuse the text-content component created in #20, and the reusable slider button in  #24. I want a good experience on the web and phone.

Design:

booklets_page_hero

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in libs/features/pages/booklets-page inside a components folder. The component's name is booklets-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by levy-web**
# Description
designed a responsive booklets page hero section that re-uses slider button compent #24, hero text component #20 and header componet #1

Reference the issue related to this PR as shown below. Every issue has it's own number you can find the issue numbers [here](https://github.com/italanta/italanta-apps/issues).

Fixes #15

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Screenshot (optional)

Screenshot from 2023-09-04 09-44-17

Screenshot from 2023-09-04 09-57-08

Screenshot from 2023-09-04 09-45-00

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/booklets-page/src/lib/components/booklets-hero-section/booklets-hero-section.component.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+export class BookletsHeroSectionComponent {
+  herotexts: HeroText = {
+    title:"Booklets",
+    description:"Your organisation has the potential to turn every task into an opportunity to learn. From growing crops, to nursing someone to good health, to developing the next-generation technology, let’s work together to make the magic happen."
+  }
+
+  buttonData: SliderButtonData = {
+    text: "Unlock your Potential"
+  }
+
```
Comment (AnnaL001) [inline libs/features/pages/booklets-page/src/lib/components/booklets-hero-section/booklets-hero-section.component.ts:20]: For the purposes of readability and maintainability and keeping the component clean, the 'herotexts' and 'buttonData' variable content can be moved to a Typescript file within the data module and referenced from there.

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Good job, a number of changes before this is merged

**Comment by levy-web** [comment]
@AnnaL001 completed the fix kindly review

**Comment by LemmyMwaura** [comment]
Work on only 2 and 3 pointers now

**Pull request closed without merging**